tools/specgen/specgen.go

```diff
@@ -140,6 +140,12 @@ var typesToDisambiguate = map[string]*CSModelType{
 	typeToKey(reflect.TypeOf(types.NetworkDisconnect{})):     {Name: "NetworkDisconnectParameters"},
 	typeToKey(reflect.TypeOf(types.NetworksPruneReport{})):   {Name: "NetworksPruneResponse"},
 	typeToKey(reflect.TypeOf(types.NetworkResource{})):       {Name: "NetworkResponse"},
+	typeToKey(reflect.TypeOf(types.PluginConfigInterface{})): {
+		Name: "PluginConfigInterface",
+		Properties: []CSProperty{
+			CSPropertyName{Name: "Types", Type: CSType{"System.Collections.Generic", "IList<string>", false}},
+		},
+	},
```

Comment:
This looks good did you forget to re-run update? I am surprised to still see the PluginInterfaceType.Generated.cs

Reply:
I tried to install go compiler on my windows machine. Apparently, the site was unreachable yesterday, and I thought this could be an independent change.